packages/compass-components/src/components/item-action-controls.tsx

```diff
@@ -567,7 +569,9 @@ export function DropdownMenuButton<Action extends string>({
             rightGlyph={<Icon glyph={'CaretDown'} />}
             title={buttonText}
           >
-            <span className={hiddenOnNarrowStyles}>{buttonText}</span>
+            <span className={hideOnNarrow ? hiddenOnNarrowStyles : undefined}>
```

Comment:
This is a drive-by. Recently I made the dropdown button collapse when the view gets narrow, but for the create index button you get when search indexes is available that's a bit much. So I added a way to opt out of it.